Makefile

```diff
@@ -71,17 +81,49 @@ lint:
 vet:
 	$(DOCKER_BUILD) '$(VET)'
 
-container: sonobuoy
+pre:
+	curl -sSL https://github.com/estesp/manifest-tool/releases/download/$(MANIFEST_TOOL_VERSION)/manifest-tool-linux-amd64 > $(MANIFEST_TOOL_DIR)/manifest-tool
```

Comment:
This doesn't work on OSX: `/var/folders/v9/30xb9zqj20nf56xbvmz6qpym0000gn/T/tmp.cdp3AjCw/manifest-tool: /var/folders/v9/30xb9zqj20nf56xbvmz6qpym0000gn/T/tmp.cdp3AjCw/manifest-tool: cannot execute binary file`

Why did you switch from `go get`?

Reply:
@liztio 
"go get" is back. Please check it.